src/gmp.c

```diff
@@ -8998,19 +8997,19 @@ results_xml_append_nvt (iterator_t *results, GString *buffer, int cert_loaded)
             assert (0);
           if (!next (&iterator))
             abort ();
+          severity = ovaldef_info_iterator_severity (&iterator);
           buffer_xml_append_printf (buffer,
                                     "<nvt oid=\"%s\">"
                                     "<type>ovaldef</type>"
                                     "<name>%s</name>"
-                                    "<family/>"
```

Comment:
Why is the familiy element removed? It seems to be empty but I don't think it belongs to this PR.

Reply:
I removed it because it's always empty and also missing for CVE results.